flux/stdlib/influxdata/influxdb/to.go

```diff
@@ -1,4 +1,353 @@
 package influxdb
 
-// TODO(jsternberg): Implement the to method in influxdb 1.x.
-// This file is kept around so it shows up in the patch.
+import (
+	"context"
+	"fmt"
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/runtime"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/stdlib/influxdata/influxdb"
+	"github.com/influxdata/flux/values"
+	"github.com/influxdata/influxdb/coordinator"
+	"github.com/influxdata/influxdb/models"
+	"github.com/influxdata/influxql"
+	"time"
+)
+
+const (
+	ToKind = "influx1x/toKind"
+	DefaultBufferSize = 5000
+)
+
+type ToOpSpec struct {
```

Comment:
right now `to` turns all string columns into keys - this is bad and should be fixed as a follow-up